### ISSUE TITLE
R!: command not found

### ISSUE THREAD

**Issue by marie-bnl** [victorbnl/build-signed-apk#7]
It looks like there's still a little problem after the build finished
```
> Task :app:assembleRelease

BUILD SUCCESSFUL in 12s
38 actionable tasks: 2 executed, 36 up-to-date
/home/runner/work/_temp/fdffee6c-3446-4e23-ae4e-a278fb2e0713.sh: line 7: R!: command not found
Error: Process completed with exit code 127.
```

_Originally posted by @acolombo25 in https://github.com/victorbnl/build-signed-apk/issues/6#issuecomment-1646877811_

**Comment by marie-bnl** [comment]
It’s very strange, I forked your repository and I can’t reproduce it (https://github.com/victorbnl/lock-screen/actions/runs/5660871195/job/15337815373).

**Comment by acolombo11** [comment]
I found where the problem comes from. The password of my keystore ends in `&R!`.
I tried with another keystore also having an ampersand in its password, and the error becomes `restOfThepasswordAfterAmpersand: command not found`

**Comment by marie-bnl** [comment]
That helps a lot, thank you. I’ll try to investigate it now.

**Comment by marie-bnl** [comment]
Fixed by df3fd6f.
